src/main/scala/codacy/plugins/test/serializers/Serializers.scala

```diff
@@ -0,0 +1,30 @@
+package codacy.plugins.test.serializers
+
+import com.codacy.plugins.api.results.Result
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/gh/codacy/codacy-plugins-test/pullRequest?prid=5569494)